E: dispcalgui: udev-rule-in-etc etc/udev/rules.d/55-Argyll.rules
N: 
N:    This package ships a udev rule and installs it under /etc/udev/rules.d,
N:    which is reserved for user-installed files. The correct directory for
N:    system rules is /lib/udev/rules.d.
N:    
N:    Refer to http://bugs.debian.org/559208 for details.
N:    
N:    Severity: important, Certainty: certain
N:    
N:    Check: files, Type: binary, udeb
Index: dispcalgui-0.9.9.1/dispcalGUI/setup.py
===================================================================
--- dispcalgui-0.9.9.1.orig/dispcalGUI/setup.py	2012-03-22 06:01:06.000000000 +0100
+++ dispcalgui-0.9.9.1/dispcalGUI/setup.py	2012-05-06 11:23:52.800744366 +0200
@@ -374,14 +374,14 @@
 							# USB and serial instruments using udev, where udev 
 							# already creates /dev/bus/usb/00X/00X devices
 							devconf_files += [
-								("/etc/udev/rules.d", [os.path.join(
+								("/lib/udev/rules.d", [os.path.join(
 									pydir, "..", "misc", "55-Argyll.rules")])
 							]
 						else:
 							# USB using udev, where there are NOT /dev/bus/usb/00X/00X 
 							# devices
 							devconf_files += [
-								("/etc/udev/rules.d", [os.path.join(
+								("/lib/udev/rules.d", [os.path.join(
 									pydir, "..", "misc", "45-Argyll.rules")])
 							]
 					else:
